tests/codegen/lift-lower-foreign.wit

```diff
@@ -37,17 +37,17 @@ world foo {
   use a.{t6 as u6, t7 as u7, t8 as %u8, t9 as u9}
   use a.{t10 as u10, t11 as u11}
 
-  export f1: func(a: u1) -> u1
-  export f2: func(a: u2) -> u2
-  export f3: func(a: u3) -> u3
-  export f4: func(a: u4) -> u4
-  export f5: func(a: u5) -> u5
-  export f6: func(a: u6) -> u6
-  export f7: func(a: u7) -> u7
-  export f8: func(a: %u8) -> %u8
-  export f9: func(a: u9) -> u9
-  export f10: func(a: u10) -> u10
-  export f11: func(a: u11) -> u11
+  export foo-f1: func(a: u1) -> u1
+  export foo-f2: func(a: u2) -> u2
+  export foo-f3: func(a: u3) -> u3
+  export foo-f4: func(a: u4) -> u4
+  export foo-f5: func(a: u5) -> u5
+  export foo-f6: func(a: u6) -> u6
+  export foo-f7: func(a: u7) -> u7
+  export foo-f8: func(a: %u8) -> %u8
+  export foo-f9: func(a: u9) -> u9
+  export foo-f10: func(a: u10) -> u10
+  export foo-f11: func(a: u11) -> u11
```

Comment:
It _is_ required, otherwise the codegen tests break at compile time.  Now that we've moved away from the `export_*` macro, we're actually exporting these symbols (i.e. not just creating a declarative macro that _would_ export the symbols if called), which means we can't export the same symbol name more than once.  That means exported symbol names must be _globally_ unique across _all_ the codegen tests.  Does that make sense?

Reply:
Ah ok so there's some other codegen tests claiming these names and this one was the unlucky one to get renamed? If so that sounds good to me 👍

Reply:
Yes, correct.